source/api/external_resources.json

```diff
@@ -23,6 +23,24 @@
   },
   "Resources": {
     "APIHandler": {
+        "Metadata": {
+            "cfn_nag": {
+                "rules_to_suppress": [
+                {
+                    "id": "W89",
+                    "reason": "This Lambda function does not need to access any resource provisioned within a VPC."
```

Comment:
The reason for each of the rule suppressions are the same. Is that intended to be the case?

Reply:
I updated the comments on the rule suppressions.